test/parallel/test-buffer-alloc.js

```diff
@@ -978,9 +984,16 @@ assert.throws(() => Buffer.from('', 'buffer'), TypeError);
   }
 }
 
-assert.throws(() => Buffer.allocUnsafe((-1 >>> 0) + 1), RangeError);
-assert.throws(() => Buffer.allocUnsafeSlow((-1 >>> 0) + 1), RangeError);
-assert.throws(() => SlowBuffer((-1 >>> 0) + 1), RangeError);
+if (common.isChakraEngine) {
```

Comment:
nit consider: Mostly we list v8 behavior before chakra, e.g. all other engineSpecificMessage usage. It may feel more consistent to list v8 behavior first here (if (!isChakraEngine) ... )?